modules/linear_boltzmann_solvers/a_lbs_solver/lbs_solver.cc

```diff
@@ -438,7 +438,7 @@ LBSSolver::OptionsBlock()
   "The eager limit is the message size limit before which non-blocking MPI send"
   "calls will execute without waiting for a matching receive call. The limit is"
   "platform dependent but in general 64 kb. Some systems have 32 kb as a limit"
-  "and therefore we use that as a default limit in ChiTech. There is a fine"
+  "and therefore we use that as a default limit in OpenSn. There is a fine"
```

Comment:
I thought the name is openSn. Did we change that?

Reply:
Old habits die hard with the capital "O". The org is "O" while the repo is "o". I will change this to "o".